parley/src/style/font.rs

```diff
@@ -83,6 +83,18 @@ impl<'a> FontFamily<'a> {
     }
 }
 
+impl<'a> Into<FontFamily<'a>> for GenericFamily {
+    fn into(self) -> FontFamily<'a> {
+        FontFamily::Generic(self)
+    }
+}
+
+impl<'a> Into<FontStack<'a>> for &'a str {
+    fn into(self) -> FontStack<'a> {
+        FontStack::Source(Cow::Borrowed(self))
+    }
+}
+
```

Comment:
I believe it's recommend to implement `From` rather than `Into` when you own both types (there's a blanket impl of Into for types that implement From).

Reply:
Addressed.